### ISSUE TITLE
Drop empty hgvs columns from table preview

### ISSUE THREAD

**Issue by afrubin** [VariantEffect/mavedb-ui#95]
The data table preview currently shows all three possible hgvs columns (hgvs_nt, hgvs_splice, hgvs_pro) even when only one is used by the dataset.

It makes sense to provide all of these columns in the csv output for consistency, but when showing the data table preview it seems reasonable to omit the unpopulated columns. hgvs_splice in particular is seldom used, and we have many datasets with only hgvs_pro variants.

**Comment by ashsny** [comment]
I suspect the reason these columns exist in the UI is because the UI is
using the same API endpoint as CSV download does

**Comment by EstelleDa** [comment]
> I suspect the reason these columns exist in the UI is because the UI is using the same API endpoint as CSV download does
> […](#)

Yes, you're correct. In fact, we have the function of showing data in table preview first. I think downloading CSV file can use it directly so I just used the function of showing data in table preview for downloading CSV file.

**Comment by afrubin** [comment]
Originally we decided to keep all of the hgvs columns in csv output to make it easier for users who are downloading a lot of datasets, but we could drop the null columns and make the users identify which columns are present.

It seems like making that change to the csv logic would solve this issue as well, right?

**Comment by bencap** [comment]
We would prefer to solve this by dropping the empty columns from the UI table view, not by changing the generated CSV.

**Comment by EstelleDa** [comment]
We changed this part in https://github.com/VariantEffect/mavedb-api/issues/230. Neither the downloaded CSV files nor the data table shows any empty hgvs columns.
